scst-scan/setup-supply-chain-component.md

```diff
@@ -0,0 +1,120 @@
+# Setup the Supply Chain Component
+
+This topic covers how to install the Trivy Supply Chain Component, create custom Supply Chain Component using the SCST - Scan 2.0, and view the components that are available to be used in the Tanzu Supply Chain.
+
+## <a id="overview"></a> Overview
+
+* [Install Trivy Supply Chain Component](./setup-supply-chain-component.md#install-trivy-supply-chain-component)
+* [Customize Scanning Component](./setup-supply-chain-component.md#customize-scanning-component)
+* [How to view the component](./setup-supply-chain-component.md#how-to-view-component)
+
+### <a id="install-trivy-component"></a> Install Trivy Supply Chain Component
+
+This section covers how to install the Trivy Supply Chain Component that uses SCST - Scan 2.0.
+
+1. List version information for the Trivy Supply Chain Component package by running:
+```
+tanzu package available list trivy.app-scanning.component.apps.tanzu.vmware.com --namespace tap-install
+```
+
+For example:
+```
+$ tanzu package available list trivy.app-scanning.component.apps.tanzu.vmware.com --namespace tap-install
+NAME                                                VERSION                              RELEASED-AT
+trivy.app-scanning.component.apps.tanzu.vmware.com  TRIVY-COMPONENT-VERSION              2024-01-26 12:35:39 -0500 EST
+```
+
+2. Install Trivy Supply Chain Component package.
+```
+tanzu package install trivy-app-scanning-component --package-name trivy.app-scanning.component.apps.tanzu.vmware.com \
+    --version TRIVY-COMPONENT-VERSION \
+    --namespace tap-install
+```
+
+
+### <a id="customize-component"></a> Customize Scanning Component
+
+This section covers how to create a custom Scanning Supply Chain Component that uses SCST - Scan 2.0.
+
+1. Customize a component by retrieving the yaml of the Trivy Supply Chain Component and modifying the following fields:
+    ```
+    kubectl get component trivy-image-scan-1.0.0 -n trivy-app-scanning-catalog -o yaml > component.yaml
+    apiVersion: supply-chain.apps.tanzu.vmware.com/v1alpha1
```

Comment:
**Suggestion**: I think adding a line break to have some separation between the command and the output (not really output but same same) will help with readability

Reply:
I decided to separate this into two segments:
* First show the command
* Then show the example of the command